jquery.selectBox.js

```diff
@@ -762,6 +786,68 @@
         }
     };
 
+    /**
+     * Handles the input event, for search in dropdowns with scroll enabled
+     *
+     */
+	SelectBox.prototype.handleInput = function () {
+		var select = $(this.selectElement)
+            , control = select.data('selectBox-control')
+            , options = control.data('selectBox-options')
+            , self = this;
+
+		if (control.hasClass('selectBox-disabled')) {
+			return;
+		}
+
+		
+		// Type to find
+		if (!control.hasClass('selectBox-menuShowing')) {
+			this.showMenu();
+		}
+		this.typeSearch = control.find('.selectBox-input').val();
+		options.find('A').each(function () {
+			if ($(this).text().substr(0, self.typeSearch.length).toLowerCase() === self.typeSearch.toLowerCase()) {
```

Comment:
In our current implementation, I had to patch this in this way:

```suggestion
			if ($(this).text().trim().substr(0, self.typeSearch.length).toLowerCase() === self.typeSearch.toLowerCase()) {
```